### PR TITLE
Metrics and lighting implementation

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: photon-server/src/main/java/org/photonvision/common/hardware/HardwareManager.java [hunk 1]
```diff
@@ -61,27 +61,46 @@ private HardwareManager(HardwareConfig hardwareConfig) {
...
     }
 
+    public void setBrightnessPercent(int percent) {
+        ConfigManager.getInstance().getConfig().getHardwareSettings().ledBrightnessPercentage = percent;
+        if (visionLED != null) visionLED.setBrightness(percent);
+        ConfigManager.getInstance().requestSave();
+        logger.info("Setting led brightness to " + percent + "%");
+    }
+
     private void onJvmExit() {
```
Comment (XzibitGG) [inline photon-server/src/main/java/org/photonvision/common/hardware/HardwareManager.java:102]: We should also call the [shutdown ](https://github.com/PhotonVision/photonvision/blob/master/photon-server/src/main/java/org/photonvision/common/hardware/GPIO/GPIOBase.java#L74) methods on the LEDs

File: photon-server/src/main/java/org/photonvision/common/hardware/metrics/CPUMetrics.java [hunk 2]
```diff
@@ -26,10 +26,13 @@ public String getMemory() {
         return execute(cpuMemoryCommand);
     }
 
-    // TODO: Command should return in Celsius
     public String getTemp() {
         if (cpuTemperatureCommand.isEmpty()) return "";
-        return execute(cpuTemperatureCommand);
+        try {
+            return String.valueOf(Double.parseDouble(execute(cpuTemperatureCommand)) / 1000);
```
Comment (XzibitGG) [inline photon-server/src/main/java/org/photonvision/common/hardware/metrics/CPUMetrics.java:32]: Converting the output of the command to a string without stripping away non-numeric characters somethings threw errors, would probably add some sort of regex before parsing.

File: photon-server/src/main/java/org/photonvision/common/hardware/HardwareManager.java (hunk 1 above)
Comment (mcm001) [reply photon-server/src/main/java/org/photonvision/common/hardware/HardwareManager.java:102]: What exactly does the shutdown method do?

File: photon-server/src/main/resources/web/index.html [hunk 3]
```diff
@@ -1 +1 @@
-<p>UI has not been copied!</p>
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/favicon.png><title>PhotonVision</title><link href=/css/app.ec4da1e6.css rel=preload as=style><link href=/css/chunk-vendors.25378257.css rel=preload as=style><link href=/js/app.801f25e3.js rel=preload as=script><link href=/js/chunk-vendors.6d469ff5.js rel=preload as=script><link href=/css/chunk-vendors.25378257.css rel=stylesheet><link href=/css/app.ec4da1e6.css rel=stylesheet></head><body><noscript><strong>We're sorry but PhotonVision doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.6d469ff5.js></script><script src=/js/app.801f25e3.js></script></body></html>
```
Comment (Bankst) [inline photon-server/src/main/resources/web/index.html:1]: Pls delet

**Review by Bankst — CHANGES_REQUESTED** [review]

**Review by Bankst — APPROVED** [review]

**Pull request merged**